src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/custom.py

```diff
@@ -334,10 +335,17 @@ def _get_site_credential(client, resource_group, name):
     return (creds.publishing_user_name, creds.publishing_password)
 
 def _stream_trace(streaming_url, user_name, password):
-    import pycurl
-    import certifi
-    c = pycurl.Curl()
-    c.setopt(c.URL, streaming_url)
-    c.setopt(pycurl.CAINFO, certifi.where())
-    c.setopt(c.USERPWD, '{}:{}'.format(user_name, password))
-    c.perform()
+    import urllib3
+
+    urllib3.disable_warnings()
+    http = urllib3.PoolManager()
+    headers = urllib3.util.make_headers(basic_auth='{0}:{1}'.format(user_name, password))
+    r = http.request(
```

Comment:
stream request will be closed when client reads the end of content or server terminates it.